tests/antares/services/api_services/test_study_api.py

```diff
@@ -250,3 +251,70 @@ def test_read_study_api(self):
             assert actual_study.name == expected_study.name
             assert actual_study.version == expected_study.version
             assert actual_study.service.study_id == expected_study.service.study_id
+
+    def test_create_variant_success(self):
+        variant_name = "variant_test"
+        with requests_mock.Mocker() as mocker:
+            base_url = "https://antares.com/api/v1"
+            url = f"{base_url}/studies/{self.study_id}/variants?name={variant_name}"
+            variant_id = "variant_id"
+            mocker.post(url, text=variant_id, status_code=201)
+
+            variant_url = f"{base_url}/studies/{variant_id}"
+            mocker.get(variant_url, json={"id": variant_id, "name": variant_name, "version": "880"}, status_code=200)
+
+            config_urls = re.compile(f"{base_url}/studies/{variant_id}/config/.*")
+            mocker.get(config_urls, json={}, status_code=200)
+
+            areas_url = f"{base_url}/studies/{variant_id}/areas?ui=true"
+            mocker.get(areas_url, json={}, status_code=200)
+
+            variant = self.study.create_variant(variant_name)
+
+            assert isinstance(variant, Study)
+            assert variant.name == variant_name
+            assert variant.service.study_id == variant_id
+
+    def test_create_variant_api_success(self):
```

Comment:
As it's the exact same code except for the part where you use the method, i think it would be easier to understand to put all inside one test to avoid duplicating code